opentelemetry/deployment/README.md

```diff
@@ -0,0 +1,25 @@
+# Deployment
+
+Overall, this configuration sets up an OpenTelemetry collector deployed as a Deployment that receives Jaeger, OTLP, OpenCensus and Zipkin traces, adds Kubernetes attributes, and exports the traces to Tempo via OTLP.
+
+## How to run
+1. Deploy the resources from the `otelcol.yaml` manifest file:
+    ```sh
+    oc apply -f otelcol.yaml
+    ```
+
+
+## Explanation
+This example configuration defines an OpenTelemetry Collector in an OpenShift environment. The collector is configured to run as a deployment in the cluster. The number of replicas will be determined by the `.spec.replicas` parameter or the parameters set in `.spec.autoscaler`.
+
+The collector is configured with a receiver for Jaeger traces over the Thrift HTTP protocol, OpenCensus traces over the OpenCensus protocol, Zipkin traces over the Zipkin protocol and OTLP traces over the GRPC protocol.
```

Comment:
I would remove ` over the Thrift HTTP protocol`  because the receiver now receives multiple formats